storage/include/rocksdb/client.h

```diff
@@ -77,7 +77,7 @@ class Client : public concord::storage::IDBClient {
     if (txn_db_) {
       // If we're using a TransactionDB, it wraps the base DB, so release it
       // instead of releasing the base DB.
-      dbInstance_.release();
+      (void)dbInstance_.release();
```

Comment:
What is the purpose of `(void)`?

Reply:
@dartdart26 , In general, a developer is not supposed to ignore the value returned from `unique_ptr::release`.
Since `clang-tidy` warns about this, I add `(void)` to suppress this warning and show that it is by design.

Reply:
Got it, thanks!